packages/liquidator/index.js

```diff
@@ -196,9 +202,37 @@ async function run({
     const gasEstimator = new GasEstimator(logger);
     await gasEstimator.update();
 
+    let dsProxyManager;
+    if (proxyTransactionWrapperConfig?.isUsingDsProxyToLiquidate) {
+      dsProxyManager = new DSProxyManager({
+        logger,
+        web3,
+        gasEstimator,
+        account: accounts[0],
+        dsProxyFactoryAddress:
+          proxyTransactionWrapperConfig?.dsProxyFactoryAddress || getAddress("DSProxyFactory", networkId),
+        dsProxyFactoryAbi: getAbi("DSProxyFactory"),
```

Comment:
Since these two ABI's are hardcoded, I think it might be cleaner to call `getAbi()` from within the DXProxyManager. I do this with the `funding-rate-proposer/src/proposer.js` client, but I'm not strongly opionated.

Reply:
Ideally I dont want to modify the `DSProxyManager` in this PR though as I feel that'll be covering too much. I'll do it in a follow up PR where I refactor the `DSProxyManager` to use the `runTransaction`.